cli/src/semgrep/lsp/server.py

```diff
@@ -70,130 +62,11 @@ def poll_ls(self) -> None:
             return
         status = self.core_process.poll()
         if status is not None and status != 0:
-            self.notify_show_message(
-                1,
-                f"OSemgrep process died with status {status}, exiting...",
-            )
```

Comment:
why remove this? can we keep it as debug logging or something?

Reply:
it's useful for users, but since this code is only for testing now it's kind of useless